pennylane_lightning/lightning_tensor/_tensornet.py

```diff
@@ -31,44 +31,100 @@
 from pennylane.wires import Wires
 
 
-def svd_split(M, bond_dim):
-    """SVD split a matrix into a matrix product state via numpy linalg. Note that this function is to be moved to the C++ layer."""
-    U, S, Vd = np.linalg.svd(M, full_matrices=False)
+def svd_split(Mat, site_shape, max_bond_dim):
+    """SVD decomposition of a matrix via numpy linalg. Note that this function is to be moved to the C++ layer."""
+    # TODO: Check if cutensornet allows us to remove all zero (or < tol) singular values and the respective rows and columns of U and Vd
+    U, S, Vd = np.linalg.svd(Mat, full_matrices=False)
     U = U @ np.diag(S)  # Append singular values to U
```

Comment:
We can use the broadcasting numpy ops to avoid the construction of the diagonal matrix 
```suggestion
    U, S, Vd = np.linalg.svd(Mat, full_matrices=False)
    U = U * S  # Append singular values to U
```